pkg/probe/http/http.go

```diff
@@ -113,6 +113,9 @@ func DoHTTPProbe(req *http.Request, client GetHTTPInterface) (probe.Result, stri
 			klog.V(4).Infof("Probe terminated redirects for %s, Response: %v", url.String(), *res)
 			return probe.Warning, fmt.Sprintf("Probe terminated redirects, Response body: %v", body), nil
 		}
+		if errors.Is(err, utilio.ErrLimitReached) {
```

Comment:
Would it be better to move Infof call here as well? Now they're separated and I understand why, but still think the code would be more readable if they're together.

Reply:
you are correct, fixed